### ISSUE TITLE
Unable to create more than 8 connections to MySQL database

### ISSUE THREAD

**Issue by sparcit** [rbraeunlich/gatling-jdbc#6]
Hi,

I am trying to send through enough load through to our RDS mysql database using jdbc-gatling but I am unable to get to more than 4 connections from a Macbook pro and not more than 8 connections from a Windows 7 machine. 
Regardless of the number of concurrent users I spawn these are maximum number of connections that are spawned.

Below I set the connectionpoolsettings to go upto 200. But it has not helped
```java
val settings = ConnectionPoolSettings(initialSize = 100, maxSize = 200)
 val jdbcConfig = jdbc.url("jdbc:mysql://blah-blah-blah")
    .username(dbusername)  // TODO Change this to perpetual or on demand Authorization
    .password(dbpassword)
    .driver("com.mysql.cj.jdbc.Driver")
    .connectionPoolSettings(settings)
    .build
```

Here are the versions of the various dependencies I am using
```xml
     <gatling.version>3.3.1</gatling.version>
      <scala-maven-plugin.version>3.3.1</scala-maven-plugin.version>
     <gatling-maven-plugin.version>3.0.5</gatling-maven-plugin.version>
      <jdbc-gatling.version>2.3.0</jdbc-gatling.version>
      <mysql-connector.version>8.0.15</mysql-connector.version>
```

**Comment by rbraeunlich** [comment]
Hey @sparcit 
that seems weird. All connection pool stuff is actually delegates to ScalikeJDBC.
When you run your simulation on DEBUG level, do you see this output somewhere?
`DEBUG scalikejdbc.ConnectionPool$ - Registered singleton connection pool : ConnectionPool(`
And does the URL match?

**Comment by sparcit** [comment]
Yes I do see that debug statement and the url matches, as I can connect of course and run the queries. It could be a bug in scalikejdbc.ConnectionPool as the connectionpool settings object gets passed in with all the parameters I set. But the maximum number opened on the Macbook is just 4
Is there a way to bypass the connection pool and open a connection for each user?

**Comment by rbraeunlich** [comment]
Hey @sparcit 
I just wanted to make sure that the default connection pool is being used and not a different one.
I checked with the debugger and the properties get properly passed on to ScalikeJDBC. There is also a unit test that covers this.
Could you try to write a simple test that directly uses ScalikeJDBC and check if you can create more than the four connections? Then we could be sure if the error is within Scalike or Gatling JDBC.
Currently, there is no way to bypass the connection pool since all database actions use the implicitly present one.

**Comment by pshingavi** [comment]
@rbraeunlich 
Was this resolved ? I see the same issue, I am only seeing 12 connections although my connection pool settings are different.
